yarn.lock

```diff
@@ -18525,9 +18525,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#8a9ef5109d833dcc65a1bf2c2ae785c3cccb1466":
-  version "18.1.0"
-  resolved "https://github.com/department-of-veterans-affairs/vets-json-schema.git#8a9ef5109d833dcc65a1bf2c2ae785c3cccb1466"
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#c12156e77d9cbffa439ee0ec2ddf42b805658023":
+  version "18.4.0"
```

Comment:
We definitely don't want to check in this change, it bumps the schema version which other forms depend on